examples/dbinit.py

```diff
@@ -0,0 +1,30 @@
+"""
+Register nodes with HaaS.
+
+This is intended to be used as a template for either creating a mock HaaS setup
+for development or to be modified to register real-life nodes that follow a
+particular pattern.
+"""
+
+from subprocess import check_call
+
+N_NODES=10
+
+ipmiUser = "ADMIN_USER"
+ipmiPass = "ADMIN_PASSWORD"
+
+def haas(*args):
+    args = map(str, args)
+    print args
+    check_call(['haas'] + args)
+
+for n in range(N_NODES):
+    node = n
+    ipmiIP = "10.0.0.0" + str(node+1)
```

Comment:
I think you want to leave off the last `0` octet. This seems to *work* as is, which isn't that surprising, but it's a little funny looking.